source/github-helpers/api.ts

```diff
@@ -29,6 +29,8 @@ import mem from 'mem';
 import * as pageDetect from 'github-url-detection';
 import {JsonObject, AsyncReturnType} from 'type-fest';
 
+import {logQuery} from '../features/index';
+
```

Comment:
Nit. Extra spance and don't forget to resort the from

Reply:
Ok, done.